### DISCUSSION TITLE
Open with... support

### DISCUSSION THREAD

**Discussion by vincilios** [sylikc/jpegview#257] [Ideas]
One feature that it missing to me is be able to right click and open the image with photoshop or other program. 
Can you add this to the context manu?
Would be great. Thanks 😁

**Comment by hrxn** [comment]
Missing feature?

Should already be possible to do...

```ini
; -----------------------------------------------
; - OPEN WITH COMMANDS
; -----------------------------------------------

; Open with menu commands, shown in the 'Open with..' submenu of the JPEGView context menu.
;
; These commands must be named OpenWith# where # stands for a number. The first open with command has the number zero (OpenWith0)
; and the numbering of the commands must be continuous.
;
; Open with commands must have the following form:
;     OpenWith#="Cmd: '%Cmd%' Menuitem: '%menu%' [KeyCode: %Key%] [Confirm: '%confirm%'] [Flags: '%flags%']"
;
; For an explanation of the different options, refer to the user command documentation above.
;
; Example for a menu entry that opens the current image in MS Paint:

;OpenWith0="Cmd: 'C:\WINDOWS\system32\mspaint.exe %filename%' Menuitem: 'Microsoft Paint' Flags: 'ShellExecute'"



```
Edit your `JPEGView.ini` accordingly.

**Comment by pureby** [comment]
Another way is to `Open with...` > `Choose another app` > (scroll to the bottom) > `More apps` > `Look for another app on this PC` > (Locate and select your `JPEGView.exe`)
Once you have have done it, JPEGView will be in the _Open with..._ menu.

If you select `Open with...` > `Choose another app` a second time, `JPEGView` will be shown in the main list. If you click on it _once,_ you will see an option like: _"Always use this app to open this filetype",_ if you select it you will make JPEGView default for that filetype.
